catalyst-gateway/bin/src/main.rs

```diff
@@ -8,6 +8,7 @@ mod db;
 mod logger;
 mod service;
 mod settings;
+mod utilities;
```

Comment:
Dont want to be picky, but maybe rename just to `utils` ? 
BTW maybe we should agree on the same naming on such modules ?